milatools/cli/commands.py

```diff
@@ -1034,32 +1034,49 @@ def _get_disk_quota_usage(
     Returns whether the quota is exceeded, in terms of storage space or number of files.
 
     Here is what the output of `disk-quota` looks like on the Mila cluster:
-    ```console
-
-    Quota information for storage pool Default (ID: 1):
+    The mila cluster uses a lustre filesystem for $HOME. Here is the command we use:
+    ```bash
+    #!/bin/sh
+    # (disk-quota)
+    echo "==== HOME ===="
+    lfs quota -h -u "${1:-$USER}" /home/mila
+
+    echo ""
+    echo "==== SCRATCH ===="
+    beegfs-ctl --cfgFile=/etc/beegfs/scratch.d/beegfs-client.conf --getquota --uid "${1:-$USER}"
+    ```
```

Comment:
That's the content of the `disk-quota` command, remove.

Reply:
Fixed in e3d7a90107f6a2e5f53741cc9425e354af5bbeed